### DISCUSSION TITLE
[Dashboard] Refine Dashboard Capabilities: unecessary `apiversion` information

### DISCUSSION THREAD

**Discussion by zzxwill** [kubevela/kubevela#822] [General] [answered]
- Please kindly remove `apiversion` information as users don't care much about it
![image](https://user-images.githubusercontent.com/2805315/103117712-add70f00-46a6-11eb-9e9c-b7368feb6dcc.png)

**Comment by sunny0826** [comment, marked as answer]
I will remve this
